aws/cloudwatch-alarm-expression/main.tf

```
@@ -40,4 +40,10 @@ resource "aws_cloudwatch_metric_alarm" "alarm" {
 resource "massdriver_package_alarm" "package_alarm" {
   display_name      = var.display_name
   cloud_resource_id = aws_cloudwatch_metric_alarm.alarm.arn
+  metric {
+    name       = var.metric_queries[var.display_metric_key].metric_name
+    namespace  = var.metric_queries[var.display_metric_key].namespace
+    statistic  = try(var.metric_queries[var.display_metric_key].stat, null)
+    dimensions = var.metric_queries[var.display_metric_key].dimensions
```

```suggestion
    dimensions = try(var.metric_queries[var.display_metric_key].dimensions, {})
```

I think there is no guarantee dimensions will be present.